src/AppInstallerCLICore/Commands/CompleteCommand.cpp

```diff
@@ -67,7 +67,7 @@ namespace AppInstaller::CLI
         }
         catch (const CommandException& ce)
         {
-            AICLI_LOG(CLI, Info, << "Error encountered during completion, ignoring: " << ce.Message());
+            AICLI_LOG(CLI, Info, << "Error encountered during completion, ignoring: " << ce.Message().get());

```

Comment:
If you are having to make this change it suggests that you broke something with the ouput operators.

Reply:
Just the changing of moving from a Singleton instance to ThreadGlobal instance has caused this issue. I am clueless as to how that could affect the logging itself. I will retry investigating this for a clue.